ecosystem/sep-0008.md

```diff
@@ -3,54 +3,65 @@
 
-**Target Audience**: Asset issuers, issuance platforms, wallet developers, and exchanges
+## Target Audience
+
+Asset issuers, issuance platforms, wallet developers, and exchanges.
 
 ## Motivation
 
-Stellar aims to be an ideal platform for issuing securities. As such, it must provide tools to comply with various regulatory requirements. Regulation often requires asset issuers to monitor and approve every transaction involving the issued asset and to enforce an assortment of constraints. This SEP provides support for these requirements.
+Stellar is an ideal platform for issuing securities. Regulation sometimes requires asset issuers to monitor and approve each transaction involving issued assets and to enforce constraints. This is possible on the Stellar network today, however there exists no standard interface between wallets and issuers to negotiate approval.
 
 ## Overview
 
-Implementing a Regulated Asset consists of these parts:
-- **Stellar.toml**: Issuers will advertise the existence of an Approval Service and the approval criteria via their stellar.toml file.
-- **Approval Server**: HTTP protocol for transaction signing.
-- **Account Setup**: Wallets will have to work with accounts that are controlled or at least partially controlled (via multisig) by asset issuers, rather than the wallet end user.<br/>
-**Note**: this step may not be required once the proposed [protocol change](https://github.com/stellar/stellar-protocol/issues/146) allowing for protocol-level per-transaction approval is implemented.
+Implementing a regulated asset consists of these parts:
+- [Identification]: Regulated asset issuers set the [authorization flags] in the asset issuing account. Regulated asset issuers advertise the existence of an approval server and the approval criteria via their [SEP-1 stellar.toml] file.
+- [Transaction Composition]: Transactions consist of operations that authorize the holding account, transact the regulated asset, and deauthorize the holding account.
+- [Approval Server]: Endpoints defined in this SEP that a client uses to obtain a signature for a transaction that temporarily authorizes the client to operate with the asset.
 
 ## Regulated Assets Approval Flow
 
-1. User creates and signs a transaction.
+1. Wallet creates and signs a transaction.
 2. Wallet resolves asset information and detects that it's a regulated asset.
+	1. Wallet can detect whether an asset is a regulated asset by checking the [authorization flags] of the asset issuer's account.
 3. Wallet sends the transaction to the approval server.
-4. Approval server determines whether the transaction is compliant based on the current state of the ledger, known pending transactions, and its compliance ruleset.
+4. Approval server determines whether the transaction should be approved.
```

Comment:
❓ It's unclear why this statement needs to change.

Reply:
This is one of the changes ported from the other PR. In fact, this is [what](https://github.com/stellar/stellar-protocol/pull/750#discussion_r512198021) you had suggested.